src/authorization-reset-all-users.ts

```diff
@@ -21,6 +21,14 @@ export const resetAllUsers = async () => {
     for (const user of users) {
       count++;
       logger.info(`[${count}] - processing user (${user.displayName})`);
+      if (
```

Comment:
this code should not be here...

Reply:
Not sure who added this code, I am assuming it is related to https://github.com/alkem-io/server/issues/1950 - so we probably need to fix that before removing this code?

Reply:
I have resolved https://github.com/alkem-io/server/issues/1950 and removed the code.